processors.py

```diff
@@ -321,6 +322,13 @@ def _next_free_dialog(self):
         c.close()
         return dialogs[-1] + 1
 
+    def _set_timestamp(self, address):
+        """Сохраняет текущую дату в last_active
+        таблицы sessions"""
```

Comment:
Well, it's the third time my comment disappears, so fine, I'll say it again. A `.timestamp()` method is to be called on an `datetime` object with no arguments, not an a class. We don't call `str.upper('string')`, the same thing here